conf/modules.config

```diff
@@ -244,14 +244,12 @@ process {
     withName: "BWA.*MEM" {
         ext.args         = { meta.status == 1 ? '-K 100000000 -M -B 3' : '-K 100000000 -M' }
         ext.prefix       = { params.split_fastq > 1 ?  "${meta.id}".concat('.').concat(reads.get(0).name.findAll(/part_([0-9]+)?/).last()) : "" }
+        publishDir       = [
+            enabled: false
+        ]
     }
 
-    withName: 'MERGE_MAPPING' {
-        ext.when         = { params.save_bam_mapped || (params.skip_tools && params.skip_tools.contains('markduplicates')) }
-    }
-
-    withName: 'INDEX_MAPPING' {
-        ext.when         = { params.save_bam_mapped || (params.skip_tools && params.skip_tools.contains('markduplicates')) }
+    withName: 'INDEX_MERGE_BAM' {
```

Comment:
why did you remove the `when` here?

Reply:
No need for it anymore

Reply:
I'll check again all `when` once done with the refactoring just to be sure all is as it should be

Reply:
ok Isee. Out of curiosity, why use an `if` statement in this case but for everything else `when`? Don't get me wrong, i still find `if` a lot easier to read, just want to learn more about this new syntax :)

Reply:
Trying to find something nice to look at, easy to read, compact and easy enough to understand, so mainly trying things out :-D

Reply:
ha, I suppose there will be many cycles until everyone is completely happy